sdk/core/core-http/lib/policies/exponentialRetryPolicy.ts

```diff
@@ -50,6 +50,42 @@ const DEFAULT_CLIENT_RETRY_COUNT = 3;
 const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 90;
 const DEFAULT_CLIENT_MIN_RETRY_INTERVAL = 1000 * 3;
 
+/**
+ * Options that control how to retry failed requests.
+ */
+export interface RetryOptions {
+  /**
+   * The maximum number of retry attempts.  Defaults to 3.
+   */
+  retryCount?: number;
+
+  /**
+   * The amount of delay in milliseconds between retry attempts. Defaults to 30000
+   * (30 seconds). The delay increases exponentially with each retry up to a maximum
```

Comment:
This is assuming that the retry mode is exponential. Shouldn't the retry options allow one to choose between fixed retry policy and the exponentially increased policy. 

The [RetryOptions for EventHubs](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-amqp/src/retry.ts#L60) takes into consideration both modes. Can we follow this model?

Reply:
On the other hand, if this options is meant only for the `exponentialRetryPolicy` then, we should rename it to `ExponentialRetryOptions`. This leaves the door open for the future when we want to add a non exponential retry policy

Reply:
Yep, it assumes exponential because `core-http` doesn't currently provide a fixed retry policy.  If that's something we'd like to add, I can add a `mode` field like `event-hubs` has with only the `exponential` option for now so we can add `fixed` later.